packages/react-composites/src/composites/ChatComposite/file-sharing/FileCard.tsx

```diff
@@ -38,23 +48,39 @@ export const FileCard = (props: FileCardProps): JSX.Element => {
   const { fileName, fileExtension, progress, actionIcon } = props;
   const theme = useTheme();
 
+  const showProgressIndicator = (): boolean => progress !== undefined && progress < 1;
+
+  const containerClassName = mergeStyles({
+    width: '14rem',
+    background: theme.palette.neutralLighter,
+    borderRadius: theme.effects.roundedCorner4,
+    boxShadow: theme.effects.elevation8
+  });
+
+  const fileInfoWrapperClassName = mergeStyles({
+    padding: '0.75rem',
+    // To make space for the progress indicator.
+    paddingBottom: showProgressIndicator() ? '0.5rem' : '0.75rem'
+  });
+
+  const actionIconClassName = mergeStyles({
+    cursor: 'pointer',
+    padding: '0.25rem'
+  });
+
+  const progressIndicatorStyles: IStyleFunctionOrObject<IProgressIndicatorStyleProps, IProgressIndicatorStyles> = {
+    itemProgress: {
+      // To make the progress indicator match the figma designs. It is a hack but works.
```

Comment:
Sorry, but I still don't understand why this is needed. Perhaps it's just my lack of understanding of CSS.

I was expecting something like: "setting the height to non-zero adds XYZ that we do not want"

We shouldn't reference figma designs in the comments / code - they're not publicly available (always), and in any case aren't versioned with the source code, so the comment isn't useful.

Reply:
This is a hack. I don't understand why this works either to be honest. :/
But this is the only way I could make it look as per the figma specifications.
Gonna modify the comment.

Reply:
With that being said, I will try to find a better solution when I implement file sharing using this component.